debian/nest.postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+rm -f /etc/profile.d/nest-simulator.sh
+touch /etc/profile.d/nest-simulator.sh
+cat >>/etc/profile.d/nest-simulator.sh <<EOF
+export NEST_INSTALL_DIR='/usr'
+export NEST_DATA_DIR='/usr/share/nest'
+export NEST_DOC_DIR='/usr/share/doc/nest'
+EOF
```

Comment:
Can't we just source `nest_vars.sh` here? That would probably be more stable in the long run.

Reply:
No. NEST was created on another system within another folder structure, so  `nest_vars.sh` does not contain the correct values.